modules/proc_manage-clairv4.adoc

```diff
@@ -0,0 +1,263 @@
+[[clair-v4]]
+= Using Clair V4 Security Scanning
+
+Clair v4 is the next generation of Clair image scanning available
+with {productname}. Clair v4 is currently released as
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview],
+which means that it is not supported for production use. However, you are
+encouraged to test Clair v4 as it represents the direction of Clair image scanning
+development.
+
+== What is Clair v4?
+
+Technically, Clair v4 is a set of micro services that can be used with {productname}
+to perform vulnerability scanning of container images associated with a set of
+Linux operating systems. The micro services design of Clair v4 makes it
+appropriate to run in a highly scalable configuration, where
+components can be scaled separately as appropriate for enterprise environments.
+The Clair v4 link:https://github.com/quay/openshift-demo/blob/master/clairv4/config.yaml[config.yaml]
```

Comment:
I'm not sure linking to the config here is that valuable. I do have this document:
https://app.lucidchart.com/invitations/accept/7c5aa5a0-6ff6-4f72-ac45-fa3669c5cb77

I believe it is public. But ideally you can export this document and make it part of the prose

Reply:
I'll cut the link to config.yaml for now. I can't get to the lucidchart document. We can revisit later.